### PR TITLE
update samples to v1

### PR DESCRIPTION
**Pull request by dizcology**
Update samples to use videointelligence v1 client library.

DO NOT MERGE until tested against the released client library.

### CODE REVIEW

File: video/cloud-client/analyze/analyze.py [hunk 1]
```diff
@@ -69,14 +67,16 @@ def analyze_explicit_content(path):
...
+        include_bounding_boxes=True)
+    context = videointelligence.types.VideoContext(
+        face_detection_config=config)
 
     operation = video_client.annotate_video(
-        path, features, video_context=context)
+        path, features=features, video_context=context)
     print('\nProcessing video for face annotations:')
 
     while not operation.done():
```
Comment (jonparrott) [inline video/cloud-client/analyze/analyze.py]: Since you're here, can you drop this while loop and just use `operation.result()`?

Comment (dizcology) [reply video/cloud-client/analyze/analyze.py]: Done.

File: video/cloud-client/analyze/analyze.py [hunk 2]
```diff
@@ -69,28 +58,24 @@ def analyze_explicit_content(path):
...
+    result = operation.result(timeout=600)
     print('\nFinished processing.')
 
     # first result is retrieved because a single video was processed
-    face_annotations = (operation.result().annotation_results[0].
-                        face_annotations)
-
-    for face_id, face in enumerate(face_annotations):
+    for face_id, face in enumerate(result.annotation_results[0].
+                                   face_annotations):
```
Comment (jonparrott) [inline video/cloud-client/analyze/analyze.py]: Please never use this style of continuation. Either do this:

```python
for face_id, face in enumerate(
        result.annotation_results[0].face_annotations):
    print('Face {}'.format(face_id))
    print('Thumbnail size: {}'.format(len(face.thumbnail)))
```

or do this (preferred in this case):


```python
faces = result.annotation_results[0].face_annotations
for face_id, face in enumerate(faces):
    print('Face {}'.format(face_id))
    print('Thumbnail size: {}'.format(len(face.thumbnail)))
```

Comment (dizcology) [reply video/cloud-client/analyze/analyze.py]: done.

**Comment by dizcology** [comment]
sample tests passed with the published client library v1.0.0.

**Pull request merged**